h/search/core.py

```diff
@@ -35,7 +35,10 @@ class Search(object):
     """
     def __init__(self, request, separate_replies=False, stats=None, _replies_limit=200):
         self.request = request
-        self.es = request.es
+        if self.request.feature('search_es6'):
+            self.es = request.es6
+        else:
+            self.es = request.es
```

Comment:
This change is the most significant part of this commit, to actually use the `search_es6` feature flag, but it isn't mentioned in the commit message.